docs/getting-started/client.md

```diff
@@ -0,0 +1,65 @@
+﻿# Client Setup
+
+Assuming you are using a valid TMP4 client
+
+1. Go to `Eternexus/root` in your client
+2. Edit the `serverinfo.py` file
+3. Replace its content with the following
+
+    ```py
+    SERVER_NAME			= "Metin2"
+    SERVER_NAME_TEST	= "Test"
+    SERVER_IP			= "127.0.0.1"
+    SERVER_IP_TEST		= "127.0.0.1"
+    CH1_NAME			= "CH1"
+    CH2_NAME			= "CH2"
```

Comment:
To keep this example file minimal we can remove CH2 - CH4 and the according ports